dev-packages/e2e-tests/test-applications/default-browser/src/index.js

```diff
@@ -0,0 +1,14 @@
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({
+  dsn: process.env.E2E_TEST_DSN,
+  integrations: [Sentry.browserTracingIntegration()],
+  tracesSampleRate: 1.0,
```

Comment:
For spans and transactions to be emitted, you also need to set `tracesSampleRate` to >0. In case of e2e tests, for consistent results, we set it to 1

Reply:
This really was the missing link! Only now do i notice that all the e2e suites that test for spans and transactions have this option enabled to 1.0.